core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -195,10 +196,22 @@ private fun dateFormatted(publishDate: Instant): String {
 }
 
 @Composable
-fun NewsResourceDate(
-    publishDate: Instant
+fun NewsResourceDateAndType(
+    publishDate: Instant,
+    resourceType: NewsResourceType
+
 ) {
-    Text(dateFormatted(publishDate), style = MaterialTheme.typography.labelSmall)
+    val date = dateFormatted(publishDate)
+
+    if (resourceType != NewsResourceType.Unknown) {
```

Comment:
Could we add a test for both possible codepaths here?

Reply:
I added a test, but I keep getting this error when running the test locally:
```
FINGERPRINT must not be null
java.lang.NullPointerException: FINGERPRINT must not be null
	at androidx.compose.ui.test.AndroidComposeUiTestEnvironment.runTest(ComposeUiTest.android.kt:280)
```

still trying to figure out why...

Reply:
ah, i had my test in `test` directory instead of `androidTest`. now it run as expected. will fix my test now that it's running.